Sums.rb

```diff
@@ -0,0 +1,19 @@
+multipleThree = Array.new
+multipleFive = Array.new
+(0..999).each do |i|
+  if i % 3 == 0
+    multipleThree << i
+
+  elsif
+   i % 5 == 0
+    multipleFive << i
+  end
+
+end
+
+threeSum = multipleThree.inject{|sum, x| sum + x}
+fiveSum = multipleFive.inject{|sum, y| sum + y}
+
+puts threeSum + fiveSum

```

Comment:
do we only care about this result, and the other two are for debugging? if so, your array function is just

```ruby
sum = 0
(0..999).each do |i|
  sum += i if i % 3 == 0 || i % 5 == 0
```